ember-file-upload/src/components/file-dropzone.ts

```diff
@@ -65,6 +65,10 @@ export default class FileDropzoneComponent extends Component<FileDropzoneSignatu
     return this.args.multiple ?? true;
   }
 
+  get allowFolderDrop() {
+    return this.args.allowFolderDrop ?? false;
+  }
+
```

Comment:
We can avoid defining a getter for this property because `undefined` is falsy anyway

```suggestion
```

Reply:
Removed